test/store_test.dart

```diff
@@ -33,7 +33,7 @@ void main() {
     });
 
```

Comment:
Consider a `tearDown()` function that closes the stream in the current store. Then there are no surprises from orphaned resources.